<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Mesh Rotation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIdentifier">
        <property name="text">
         <string>Identifier:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditIdentifier"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMeshCoordinates">
        <property name="text">
         <string>Mesh coordinates:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMeshCoordinates">
        <property name="text">
         <string>coordinates</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDatapointCoordinates">
        <property name="text">
         <string>Datapoint coordinates:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditDatapointCoordinates">
        <property name="text">
         <string>coordinates</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelNormalFrom">
        <property name="text">
         <string>Normal from</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelNormalFromX">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalFromX">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelNormalFromY">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalFromY">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelNormalFromZ">
        <property name="text">
         <string>z:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalFromZ">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelNormalTo">
        <property name="text">
         <string>Normal to</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelNormalToX">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalToX">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelNormalToY">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalToY">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelNormalToZ">
        <property name="text">
         <string>z:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNormalToZ">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999.999900000053458</double>
        </property>
        <property name="maximum">
         <double>999999.999900000053458</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
